docs/changelog.md

```diff
@@ -175,6 +175,7 @@ permalink: /changelog/
 
 * **support-ktx**
   * Added `Context.hasCamera()` to check if the device has a camera.
+  * Removed `String.toUri()` in favour of Android Core KTX.
```

Comment:
Should it be marked as Breaking change (as updating to new version will lead to compile-time errors)?

Reply:
Oh, and seems like it's need to be moved to `0.56.0` version.

Reply:
> Should it be marked as Breaking change (as updating to new version will lead to compile-time errors)?

Yes, also we normally use this format to be more noticeable, for consumers:
⚠️ **This is a breaking change:**